src/cdk/tree/padding.ts

```diff
@@ -26,21 +26,21 @@ export class CdkTreeNodePadding<T> implements OnDestroy {
 
   /** The level of depth of the tree node. The padding will be `level * indent` pixels. */
   @Input('cdkTreeNodePadding')
+  get level(): number { return this._level; }
   set level(value: number) {
     this._level = coerceNumberProperty(value);
     this._setPadding();
   }
-  get level(): number { return this._level; }
   _level: number;
 
   /** The indent for each level. Default number 40px from material design menu sub-menu spec. */
   // TODO(tinayuangao): Make indent working with a string with unit, e.g. 10em
```

Comment:
@tinayuangao Not related to this PR, but can't this be done using `coerceCssPixelValue`?

Reply:
`coerceCssPixelValue` got in recently so it probably wasn't there when the tree was implemented.